src/features/Chat/components/WelcomeWindow/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import styled, { css } from 'styled-components';
+import { palette } from '@/components/variables';
+import Text from '@/components/Text';
+import { TextButton } from '@/components/Buttons';
+
+const WelcomeWindow = () => {
+  const navigate = useNavigate();
+  const { t } = useTranslation('chat');
+
+  return (
+    <div>
+      <UpperWelcomeContainer>
+        <WelcomeText isHeader variant="h2">
+          {t('welcome.upper.title')}
+        </WelcomeText>
+        <WelcomeText>{t('welcome.upper.description')}</WelcomeText>
+      </UpperWelcomeContainer>
+      <LowerWelcomeContainer>
+        <WelcomeText isHeader variant="h2">
+          {t('welcome.lower.title')}
+        </WelcomeText>
+        <WelcomeText>{t('welcome.lower.description')}</WelcomeText>
+        <SearchButton onClick={() => navigate('/mentors')}>
+          {t('welcome.lower.button')}
+        </SearchButton>
+      </LowerWelcomeContainer>
+    </div>
+  );
+};
+
+const UpperWelcomeContainer = styled.div`
```

Comment:
I would just remove the Welcome-from these containers to make more readable names.
UpperContainer, MiddleContainer, LowerContainer etc

Reply:
It already has the Welcome in the component name now!

Reply:
Remnant from the past again.